### PR TITLE
cleanup: enable revive(blank-imports/indent-error-flow rule) checking

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
cleanup: enable revive(blank-imports/indent-error-flow rule) checking

**Which issue(s) this PR fixes**:
Part of https://github.com/karmada-io/karmada/issues/4490

**Special notes for your reviewer**:
none

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by RainbowMango** [comment]
You might need to resolve the conflict.

File: pkg/controllers/cronfederatedhpa/cronfederatedhpa_handler.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 	"fmt"
 	"sync"
 	"time"
-	_ "time/tzdata"
+	_ "time/tzdata" // import tzdata to support time zone parsing
```
Comment (RainbowMango) [inline pkg/controllers/cronfederatedhpa/cronfederatedhpa_handler.go:20]: We need to describe why this blank import is required, including who will use it.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm

**Comment by RainbowMango** [comment]
Don't know why the test was not triggered, I can't help, you might need to retry by another push.

**Comment by jwcesign** [comment]
/cc @RainbowMango 
CI passed

File: pkg/webhook/cronfederatedhpa/validating.go [hunk 2]
```diff
@@ -19,10 +19,9 @@ import (
 	"math"
 	"net/http"
 	"time"
-	_ "time/tzdata"
+	_ "time/tzdata" // import tzdata to support time zone parsing, this is needed by function time.LoadLocation
 
 	"github.com/adhocore/gronx"
-	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
```
Comment (RainbowMango) [inline pkg/webhook/cronfederatedhpa/validating.go:25]: nice.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**